test/module/irohad/multi_sig_transactions/transport_test.cpp

```diff
@@ -101,22 +148,8 @@ TEST_F(TransportTest, SendAndReceive) {
             });
         return result;
       }));
-
-  auto transport =
-      std::make_shared<MstTransportGrpc>(std::move(async_call_),
-                                         std::move(tx_factory),
-                                         std::move(parser_),
-                                         std::move(batch_factory_),
-                                         std::move(tx_presence_cache_),
-                                         completer_,
-                                         my_key_.publicKey(),
-                                         getTestLogger("MstState"),
-                                         getTestLogger("MstTransportGrpc"));
-  transport->subscribe(mst_notification_transport_);
-
   std::mutex mtx;
   std::condition_variable cv;
```

Comment:
Mutex and condition variable can be removed, since there is no more multi-threading in the test.

Reply:
You are right, they are redundant. Already removed.